invenio_base/wsgi.py

```diff
@@ -42,9 +42,8 @@ def create_wsgi_factory(mounts_factories):
         factory.
     """
     def create_wsgi(app):
-        mounts = dict(
-            [(mount, factory())
-             for mount, factory in mounts_factories.items()]
-        )
+        mounts = {}
```

Comment:
you can use dict comprehension: `mounts = {(mount, factory()) for mount, factory in mounts_factories.items()}`